README.md

```diff
@@ -81,7 +83,7 @@ node_pools = [
     initial_node_count  = 1
     min_node_count      = 1
     max_node_count      = 1
-    version             = "1.13.7-gke.24"
+    version             = "1.14.10-gke.34"
```

Comment:
Maybe you could go with tests and example on 1.15.x ? As cert-manager is using <1.15 as legacy and also Google will not support it probably soon.

Reply:
makes sense 👍